compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -67,6 +67,18 @@ DataFormat get_data_format(loco::Node *node)
 
 bool has_data_format(loco::Node *node) { return node->annot<DataFormatAnnotation>() != nullptr; }
 
+bool has_dynamic_shape(const loco::Node *node)
+{
+  const auto circle_node = loco::must_cast<const luci::CircleNode *>(node);
+  const auto shape_signature = circle_node->shape_signature().as_vector();
+  for (auto ss : shape_signature)
+  {
+    if (ss == -1)
+      return true;
+  }
+  return false;
```

Comment:
Because of #5501, shape signature will be deprecated.
Please use following codes :)

```suggestion
  for (uint32_t i = 0;i < circle_node->rank(); ++i)
    if (!circle_node->dim(i).known())
      return true;
  return false;
```

Reply:
Thank you for the instruction :) I've updated the code.